includes/admin/class-wcmypa-admin.php

```diff
@@ -237,7 +237,12 @@ public function renderVariationCountryOfOriginField(int $loop, array $variationD
                 'id'            => self::META_COUNTRY_OF_ORIGIN_VARIATION . "[{$loop}]",
                 'name'          => self::META_COUNTRY_OF_ORIGIN_VARIATION . "[{$loop}]",
                 'type'          => 'select',
-                'options'       => (new WC_Countries())->get_countries(),
+                'options'     => array_merge(
+                    [
+                        null => __('Default', 'woocommerce-myparcel'),
```

Comment:
Ik denk niet dat mensen snappen wat dit is, misschien "Shop home country" van maken of iets dergelijks? Misschien hoe woocommerce dat noemt.

En je => is niet uitgelijnd hierboven :P

Reply:
`Shop home country` is het eigenlijk ook niet. 
Er zal namelijk niet naar de store instellingen gekeken worden. Het heeft meer te maken met de instellingen die je bij de defaultCountryOfOrigin heb ingevuld (MyParcel instelling)